lib/typhoeus/request/callbacks.rb

```diff
@@ -18,7 +18,7 @@ class Request
     #   request.on_complete { |response| p 2 }
     #   request.on_complete.clear
     #   request.execute_callbacks
-    #   #=> []
+    #   #=> nil
```

Comment:
I think that `#execute_callbacks` will still return `[]` since `#each` returns the array.